tool/matomo/lang/en/watool_matomo.php

```diff
@@ -28,6 +28,8 @@
 $string['pluginname'] = 'Matomo Analytics (formerly Piwik)';
 $string['siteurl'] = 'Analytics URL';
 $string['siteurl_help'] = 'Enter your Matomo URL without http(s) or a trailing slash';
+$string['piwikurl'] = 'Piwik.js URL';
+$string['piwikurl_help'] = 'Enter your piwik.js URL without http(s) or a trailing slash, will be the same as Analytics URL if you leave it empty';
```

Comment:
I would make this much more clear what the intent is, I'll clearly label is with 'alternate' and say that in the help that it is optional and used for when you want to host your own version of the client code